src/main/java/com/spotify/avro/mojo/SchemaParserBuilder.java

```diff
@@ -0,0 +1,53 @@
+package com.spotify.avro.mojo;
+
+import org.apache.avro.Schema;
+
+import java.util.Map;
+
+public class SchemaParserBuilder {
```

Comment:
Do you think it would be valuable to extract pure interface for this one and provide default implementation class?
This is intended to be overwritten in case of need, so that would make inheritance more clean for users..
// I can do it after merge..